TAF/utils/scripts/manual/shutdown.sh

```diff
@@ -1 +1,21 @@
+# This script is to shutdown edgex after executing test cases
+
 #!/bin/sh
+
+cd ${WORK_DIR}/../edge-linux-test-pytest
+GLOBAL_VAR_DIR=${WORK_DIR}/TAF/config/global_variables.py
+
+if [ -f kvm_data.json ]
+then
+    pid=`jq '.pid' kvm_data.json`
+    kill -9 ${pid}
```

Comment:
I am guessing we never figured out why "kill -2" does not work when run from this script?  Maybe timebox the next investigation to an hour.  Make sure to:

A) Make sure the command is correct.
B) Make sure the output is verbose.
C) Check the output text from the command and see if it gives you any guidance on what happened.
D) Check the return code from the command and see what it returns

Reply:
@markabrezovec-eaton I have checked return status of "kill -2" command. but it is 0 so it not showing any error or executing command. and I also checked output of command but I did not find any guidance.

Reply:
ok